src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Views/RemoteClient/Create.cshtml

```diff
@@ -9,14 +9,47 @@
         <span asp-validation-for="ClientName" class="text-danger"></span>
     </div>
 
-    <div class="mb-3" asp-validation-class-for="ApiKey">
-        <label asp-for="ApiKey">@T["Api Key"]</label>
-        <input asp-for="ApiKey" type="password" class="form-control" />
-        <span asp-validation-for="ApiKey" class="text-danger"></span>
+    <div class="mb-3">
+        <div asp-validation-class-for="ApiKeySecret">
+            <label asp-for="ApiKeySecret">@T["Api Key Secret"]</label>
+            @await Component.InvokeAsync(
+            "SelectSecret",
+            new
+            {
+            secretType = "TextSecret",
+            selectedSecret = Model.ApiKeySecret,
+            htmlId = Html.IdFor(m => m.ApiKeySecret),
+            htmlName = Html.NameFor(m => m.ApiKeySecret),
+            required  = false,
+            })
```

Comment:
Also in the other places.

```suggestion
            new
            {
                secretType = "TextSecret",
                selectedSecret = Model.ApiKeySecret,
                htmlId = Html.IdFor(m => m.ApiKeySecret),
                htmlName = Html.NameFor(m => m.ApiKeySecret),
                required  = false,
            })
```

Reply:
Yes I know but what I'm used to do is a ctrl+K+D formatting, but in razor views the tool doesn't work well so each time we do some change we have to redo the formatting manually, what you think?

Reply:
For you if you auto-format the original indentation is created?

Reply:
Yes but no problem, for now I'm applying the suggestion

Reply:
Hmm, I just tried it in VS 17.8.0 and auto-formatting produces the indented version, even if I try to break it.